packages/gestalt/src/AvatarGroupSubcomponents.js

```diff
@@ -0,0 +1,205 @@
+// @flow strict
```

Comment:
I'm confused by the organization of this file, AvatarFoundation, and DefaultAvatar. I'm also always wary of "kitchen sink" files like this, as they tend to accumulate cruft and make it difficult to identify and delete dead code in the future.

I think we're also hitting the limits of our flat directory structure with the number and complexity of components we have now. Consider creating an Avatar folder for all these helpers, and try to keep things to one component per file (with a types or consts file for shared types/consts). We can continue to keep the exported components at the top level, but we don't need these helpers cluttering up the main directory.

Reply:
Happy to break down, I didn't because I "felt" I would add many mini-files.

Reply:
Makes sense. Don't be afraid of small, focused files — it makes them easier to remove in the future! 🙂